src/plugins/intel_cpu/tests/functional/single_layer_tests/classes/eltwise.cpp

```diff
@@ -83,6 +85,11 @@ void EltwiseLayerCPUTest::SetUp() {
     ov::test::utils::OpType opType;
     ov::AnyMap additionalConfig;
     std::tie(shapes, eltwiseType, secondaryInputType, opType, netType, inType, outType, targetDevice, additionalConfig) = basicParamsSet;
+    // we have to change model precision as well, otherwise inference precision won't affect single-node graph
+    // due to enforce inference precision optimization for the eltwise as first node of the model
+    if (ov::element::Type(netType).is_real() && additionalConfig.count(ov::hint::inference_precision.name())) {
```

Comment:
Another option would be to disable optimizations in EnforceInferencePrecision method for single layer tests